cl/assets/templates/includes/htmx_pagination.html

```diff
@@ -0,0 +1,61 @@
+{% load humanize %}
+
+{% if results.has_previous or results.has_next %}
+<div class="well v-offset-above-3 hidden-print">
+  <div class="row">
+    <div class="col-xs-2 col-sm-3">
+      {% if results.has_previous %}
+        <div class="text-left">
+          {% if webhooks %}
```

Comment:
Correct me if I'm wrong, but I think this is only used by webhooks now, right? If so, can we simplify this by removing these if/then statements?

Reply:
Clean the htmx_pagination.html file so that only webhook code remains, removing conditionals?

Reply:
Yep, exactly. It's only being used by webhooks, so why do we need the `if webhooks` junk and the unused `else`'s, right?